homeassistant/components/lacrosse_view/sensor.py

```diff
@@ -116,21 +148,31 @@ def __init__(
     ) -> None:
         """Initialize."""
         super().__init__(coordinator)
+
         sensor = self.coordinator.data[int(description.key)]
```

Comment:
If we can store the sensor instance, why not pass that to the entity and use the `sensor_id` for the unique_id? Then we don't need the sensor index at all.